Purchases/Public/RCPurchases.h

```diff
@@ -130,6 +130,26 @@ NS_SWIFT_NAME(Purchases)
 + (instancetype)configureWithAPIKey:(NSString *)APIKey
                           appUserID:(NSString * _Nullable)appUserID
                        userDefaults:(NSUserDefaults * _Nullable)userDefaults;
+
+
+/**
+ Configures an instance of the Purchases SDK with object with a custom userDefaults. Use this constructor if you want to sync status across a shared container, such as between a host app and an extension. The instance of the Purchases SDK will be set as a singleton. You should access the singleton instance using [RCPurchases sharedPurchases]
```

Comment:
`Configures an instance of the Purchases SDK with object with a custom userDefaults.`
Typo.

Reply:
Typo is here still.